lib/dao.js

```diff
@@ -2824,6 +2824,14 @@ function(data, options, cb) {
 
   var allowExtendedOperators = connector.settings &&
     connector.settings.allowExtendedOperators;
+  if (options.hasOwnProperty('allowExtendedOperators')) {
+    allowExtendedOperators = options.allowExtendedOperators === true;
+  } else if (allowExtendedOperators !== false && Model.settings.mongodb &&
+    Model.settings.mongodb.hasOwnProperty('allowExtendedOperators')) {
```

Comment:
Accessing a connector-specific configuration in generic juggler code is violating the current design. Is there any particular reason why you would like to scope `allowExtendedOperators` to MongoDB only, as opposed to having it as a top-level `Model.settings.allowExtendedOperators`?